systest/cluster/cluster.go

```diff
@@ -43,6 +49,19 @@ func WithKeys(n int) Opt {
 	}
 }
 
+// Reuse will try to recover cluster from the given namespace, if not found
+// it will create a new one.
```

Comment:
what does reuse entail? i see you saved accounts in the configmap. how about mesh state?

Reply:
reuse means that it will recover (rediscover) existing cluster, for example that was deployed as a part of the previous test run